src/components/atoms/menuItem/MenuItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./MenuItem.module.scss";
+
+interface MenuItemProps {
+  name: string;
+  ingredients: string[];
```

Comment:
It would be more flexible if the ingredient is a string instead of an array. Could you review the UI/UX design?

Reply:
> It would be more flexible if the ingredient is a string instead of an array. Could you review the UI/UX design?

my bad:(